lib/rubocop/cop/betterment/not_using_rswag.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      class NotUsingRswag < Base
+        MSG = 'Request specs must use rswag to test and document their API endpoints. See: https://github.com/rswag/rswag'
+
+        def on_new_investigation
+          return if processed_source.ast.nil?
+
+          unless valid_path_structure?(processed_source.ast)
+            add_offense(processed_source.ast)
+          end
+        end
```

Comment:
Instead of hooking into `on_new_investigation`, could we use `on_send` and match the `path` method call?

```ruby
def on_send(node)
  add_offense(node) if path_node?(node) && !method_and_response_nodes?(node)
end
```

Reply:
On second thought, maybe this should be `def on_block(node)`, but still.